content/docs/reference/pkg/python/pulumi_policy/_index.md

```diff
@@ -10,5 +10,11 @@ notitle: true
 <section id="pulumi-policy">
 <h1>Pulumi Policy<a class="headerlink" href="#pulumi-policy" title="Permalink to this headline"></a></h1>
 <div class="toctree-wrapper compound">
+<ul>
+<li class="toctree-l1"><a class="reference internal" href="deserialize/">deserialize</a></li>
+<li class="toctree-l1"><a class="reference internal" href="policy/">policy</a></li>
+<li class="toctree-l1"><a class="reference internal" href="proxy/">proxy</a></li>
+<li class="toctree-l1"><a class="reference internal" href="version/">version</a></li>
```

Comment:
These modules are supposed to be implementation details. Here's an alternative PR that addresses the problem without exposing the modules: https://github.com/pulumi/docs/pull/8224